### DISCUSSION TITLE
EMU support

### DISCUSSION THREAD

**Discussion by ghost** [github/gh-gei#1117] [Q&A] [answered]
Hi,

Does gh-gei support migrating from GHEC to EMU?  I've tried for a few days and get the below error using org admin PATs in src/dest.  PATs work fine, no network blocks.

It'll create the destination repo with a single issue, containing the error.

[ERROR] Git source migration failed. Error message: Unable to connect to the target repository. This usually happens when GitHub.com is experiencing issues - please try again in 30 minutes. If this happens repeatedly, please contact GitHub Support.

**Comment by timrogers** [comment, marked as answer]
Thanks for reaching out! I can confirm that **this tooling does support migrations from GHEC to EMU**.

It does sound like you're running into an IP allowlisting or similar issue. Do you, by any chance, use conditional access policies (CAP) with your identity provider (IdP)?

Would you be able to create a ticket with GitHub Support to enable us to look into this? In that ticket, it would be great if you could include the migration ID - either the one starting with `RM_` or the GUID.

**Reply by ghost** [reply]
Thank you for the amazingly prompt reply Tim!

RM_kgDaACRmZDk3ZGI1MS01Mzk1LTQzYmYtYmRjNC1mMGZiY2VhZmNlYmM

And, yes, we do use CAP and I've opened a ticket with support too (2346424)

Will sync with my IdP team to look there, sounds like that's where the smoking gun may be.

**Reply by timrogers** [reply]
I've double checked on our side and it's definitely a CAP problem.
